app/controllers/space/base_channel.js

```diff
@@ -28,17 +29,8 @@ export default Controller.extend({
   },
 
   actions: {
-    sendMessage: function(newMessage) {
-      let message = this.createMessage(newMessage, 'message-chat');
-
-      this.get('coms').transferMessage(
-        this.get('space.model'),
-        this.get('model.sockethubChannelId'),
-        message.get('content')
-      );
-
-      this.get('model.messages').pushObject(message);
-      this.set('newMessage', null);
+    sendMessage: function() {
+      assert('Please define "sendMessage"');
```

Comment:
This is just a base class. The classes that inherit from this (e.g. `Channel` and `UserChannel` ) need to implement the actual method. This throws an error with the hint in case you inherit without defining the method yourself.